crates/dyn-abi/src/dynamic/error.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{DynSolType, DynSolValue, Error, Result};
+use alloc::vec::Vec;
+use alloy_primitives::{keccak256, Selector};
+
+const PANIC_SELECTOR: Selector = Selector::new([0x4e, 0x48, 0x7b, 0x71]);
+const REVERT_SELECTOR: Selector = Selector::new([0x08, 0xc3, 0x79, 0xa0]);
```

Comment:
Can we add some doc comments with the string version of these? (`Panic(uint256)` & `Error(string)`)

Reply:
Could also be `alloy_sol_types::Revert::SELECTOR`